### PR TITLE
moving workspace switcher to right side

### PR DESCRIPTION
**Pull request by placegraphichere**
- changed workspace icon to Ps icon
- added icon and jsx to right side
- removed from left side

### CODE REVIEW

**Comment by iwehrman** [comment]
For the merely curious: 
image

File: src/js/jsx/PanelSet.jsx [hunk 1]
```diff
@@ -281,6 +291,14 @@ define(function (require, exports, module) {
                     <div ref="panelSet"></div>
                 );
             }
+        },
+        /**
```
Comment (iwehrman) [inline src/js/jsx/PanelSet.jsx]: Need an extra newline to separate this header from the previous function.

File: src/js/jsx/PanelSet.jsx [hunk 2]
```diff
@@ -281,6 +291,14 @@ define(function (require, exports, module) {
...
+         * Close Design Space
+         *
+         * @private
+         */
+        _handleBackToPSClick: function () {
+            this.getFlux().actions.menu.native({ commandID: 5999 });
```
Comment (iwehrman) [inline src/js/jsx/PanelSet.jsx]: Note to self: add a higher-level action for this. Note to @placegraphichere: ignore this note to self.

File: src/style/shared/icons.less [hunk 3]
```diff
@@ -55,8 +55,11 @@
 
 .toolbar__backToPs.button-simple {
     position: absolute;
-    bottom: 2rem;
+    bottom: 0;
     width: 5rem;
+    svg{
```
Comment (iwehrman) [inline src/style/shared/icons.less]: Space between `svg` and the brace.

**Comment by iwehrman** [comment]
Looks great! Just a couple little whitespace nits and we're good to go.

**Comment by placegraphichere** [comment]
Should be ready for another pass @iwehrman

**Comment by iwehrman** [comment]
It takes a village.

**Pull request merged**